app/components/Thread.tsx

```
@@ -202,13 +254,20 @@ export default function Post({
-        <SanitizedUserContent
+        <div
           className={classNames(styles.content, userContentStyles.userContent)}
-          html={content}
-        />
+          onClick={(e) => {
+            // Stop propagation if clicking on a link
+            if ((e.target as HTMLElement).tagName === "A") {
+              e.stopPropagation();
+            }
+          }}
+        >
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)